doc/proposals/hugo-doc-build.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Build and Host Documentation
+authors:
+  - "@asmacdo"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2019-10-29
+last-updated: 2019-10-29
+status: TBD
+---
+
+# Build and Host Documentation
+
+## Release Signoff Checklist
+
+- \[ \] Enhancement is `implementable`
+- \[ \] Design details are appropriately documented from clear requirements
+- \[ \] Test plan is defined
+- \[ \] Graduation criteria for dev preview, tech preview, GA
+- \[ \] User-facing documentation is created in [operator-sdk/doc][operator-sdk-doc]
+
+## Motivation
+
+Hosted documentation will improve the Operator SDK user experience by
+increasing the visibility and accessibility of the content. 
+
+## Summary
+
+This enhancement proposes a technology stack that will be used to write,
+build, and host the documentation for the Operator SDK project.
+
+Hugo will be used for static site generation from Markdown. Neflify will
+be used to host the site. 
+
+Highlights:
+  - Files will remain markdown, few syntax changes necessary.
+  - Stack is aligned with current Golang/Kubernetes community.
+  - Proof of concept: https://operator-sdk.netlify.com/
+
+## Open Questions (optional)
+
+- What domain should we use? Available options include operator-sdk.com,
```

Comment:
I swear I made this comment the first time around, but maybe I forgot to click the button.

@dmesser Do you have any input on this? It seems to me that we should coordinate plans with OLM.